fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -19,14 +19,29 @@ def self.run(params)
           command_prefix,
           'agvtool',
           'what-marketing-version',
-          '-terse1'
```

Comment:
Hi @asfalcone, this change was intentional and here is the basic reasoning:

The ```-terse1``` argument returns the version of the first info plist file it finds -- alphabetically, I pressume. With a single production target application this works as any test targets follow it alphabetically, at least in the general case. 

With the ```-terse``` argument, I currently grab the last element of the array containing the info plists:
```
if scheme.empty?
	line = results.last
else
```

To ensure backwards compatibility, I could just grab the first element of the array:
```
if scheme.empty?
	line = results.first
else
```

Either way, without specifying a scheme, this has the potential to return incorrect results unless you know the order in which the ```-terse1``` argument will return the info plists.